app/controllers/api/payment/braintree_controller.rb

```diff
@@ -29,15 +29,24 @@ def webhook
     render json: {success: true}
   end
 
+  def delete_payment_method
+    # TODO: Delete payment method on Vault, and if that's successful, destroy it locally.
```

Comment:
I don't think adding this endpoint is a good idea if it's not going to actually delete them on Braintree, it just exposes us to orphaning stuff on BT unnecessarily. I can't really think of a use case for it as it stands, and it makes me nervous cause it's unauthenticated.